mem_windows.go

```diff
@@ -25,27 +25,43 @@ func (ca *Cagent) MemResults() (MeasurementsMap, error) {
 
 	memStat, err := mem.VirtualMemoryWithContext(ctx)
 
-	results["free_B"] = nil
-	results["shared_B"] = nil
-	results["buff_B"] = nil
+	results = map[string]interface{}{
+		"total_B":           nil,
+		"free_B":            nil,
+		"free_percent":      nil,
+		"cached_B":          nil,
+		"cached_percent":    nil,
+		"used_B":            nil,
+		"used_percent":      nil,
+		"available_B":       nil,
+		"available_percent": nil,
+	}
 
 	if err != nil {
```

Comment:
Same as before, please return early in error case and omit `else` branch.

Reply:
fixed